### ISSUE TITLE
Prettify markdown files

### ISSUE THREAD

**Issue by matentzn** [OBOAcademy/obook#256]
@nicolevasilevsky we could reach out to @cthoyt to show us how to do what he did for OBO Foundry. This seems to standardise the representation of markdown. Just be careful that indentation continues to be four spaces, not two.

https://github.com/OBOFoundry/OBOFoundry.github.io/pull/2052/files

**Comment by cthoyt** [comment]
No need to reach out, that PR explains where I left the docs :)

**Comment by matentzn** [comment]
Let me rephrase: Try to run the following command after installing [`prettier`](https://prettier.io):

```shell
npx prettier --write .
```
